### PR TITLE
Tooltip component

### PR DESCRIPTION
**Pull request by se7entyse7en**
Creates a `Tooltip` component. The styling is out of the scope of this PR.

### CODE REVIEW

**Comment by se7entyse7en** [comment]
@znegrin related to what @warenlg mentioned, this PR isolates more the component to be style for the tooltips.

See [here](https://github.com/athenianco/athenian-webapp/pull/164/files#diff-5d2e4b9a3401825c053f7b5c6a195b52R9).

If you wanna see it in action, this is already attached to the 5 summary metrics charts as showed [here](https://github.com/athenianco/athenian-webapp/pull/162#issue-396413847).

File: src/js/components/charts/Tooltip.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+export default ({value, ...props}) => {
+    // TODO: This needs to be styled.
+    // `value` is a flat key-value object
+    return (
+        <Hint {...props} value={value}>
+          <div style={{background: 'black'}}>
+            <h4>Value of hint</h4>
+
+            {
+                _(value).map((v, k) => {
```
Comment (se7entyse7en) [inline src/js/components/charts/Tooltip.jsx]: @znegrin @warenlg for now this is very dummy and just creates `p` tag for each key-value.

File: src/js/components/charts/Tooltip.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+export const onValueChange = (datapoint, eventType, current, setCurrent) => {
+    console.log(`START: ${eventType}`);
+
+    if (!current ||
+        (current && (datapoint.x.toString() !== current.x.toString() ||
+                          datapoint.y !== current.y))) {
+        console.log('setting datapoint');
+        setCurrent(datapoint);
+    }
```
Comment (dpordomingo) [inline src/js/components/charts/Tooltip.jsx]: `!current || current` is redundant, also, I'm not sure why is it needed the `toString()`; I'd purpose the more simple alternative:
```suggestion
  if (current && datapoint.x === current.x && datapoint.y === current.y) {
    return
  }
  
  console.log('setting datapoint');
  setCurrent(datapoint);
```

File: src/js/components/charts/Tooltip.jsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+    console.log(`END: ${eventType}`);
+};
+
+export const onValueReset = (datapoint, eventType, current, setCurrent) => {
```
[the remark this answers is not in the document]
Comment (se7entyse7en) [reply src/js/components/charts/Tooltip.jsx]: for now that's the only thing + logging, but I'd rather keep bounded here together the logic for the tooltip.

File: src/js/components/charts/Tooltip.jsx (hunk 2 above)
Comment (se7entyse7en) [reply src/js/components/charts/Tooltip.jsx]: the `toString` was necessary because wasn't working for dates. btw this has been fixed already in next PR

**Pull request merged**